src/main/controllers/features/SoleTrader/soleTraderCorrespondenceAddressManualController.ts

```diff
@@ -5,9 +5,9 @@ import * as config from "../../../config";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
     req.session.user = req.session.user || {};
-    res.render(config.SOLE_TRADER_MANUAL_CORRESPONDANCE_ADDRESS, {
-        title: "What is your correspondance address?",
-        previousPage: "/sole-trader/correspondance-address--lookup",
+    res.render(config.SOLE_TRADER_MANUAL_CORRESPONDENCE_ADDRESS, {
+        title: "What is the correspondence address?",
+        previousPage: "/sole-trader/correspondence-address--lookup",
```

Comment:
These can go in the urls folder in pageURLs.ts

Reply:
addressed the comments